galite-core/src/main/kotlin/org/kopi/galite/form/dsl/Form.kt

```diff
@@ -255,7 +272,20 @@ abstract class Form : Window() {
     }.toTypedArray()
 
     //TODO ----------begin-------------
-    this.commands = arrayOf()
+    /** Used actors in form*/
+    val usedActors = actors.map { vActor ->
+      vActor?.actorIdent to vActor
+    }.toMap()
```

Comment:
I dont think it's a good idea to create a map. Find another way to get the command's actor